src/wilcoxon.jl

```diff
@@ -129,10 +129,13 @@ function pvalue(x::ExactSignedRankTest; tail=:both)
     end
 end
 
+ci(x::ExactSignedRankTest, alpha::Float64=0.05; tail=:both) = calculate_ci(x.vals, alpha; tail=tail)
+
 
 ## APPROXIMATE SIGNED RANK TEST
 
 immutable ApproximateSignedRankTest{T<:Real} <: HypothesisTest
+    vals::Vector            # original values
```

Comment:
annoyingly changing this `Vector{T}` causes issues with the call from https://github.com/JuliaStats/HypothesisTests.jl/blob/master/src/wilcoxon.jl#L155

going to dig into why this is the case tomorrow but if anyone has any ideas let me know